src/interface/src/app/types/data.types.ts

```diff
@@ -3,24 +3,28 @@ export interface ConditionsConfig {
   filepath?: string;
   region_name?: string;
   pillars?: PillarConfig[];
+  colormap?: string;
```

Comment:
Not in this PR, but for a follow-up: can we add a REST endpoint to get the actual color values associated with a colormap name? We'll need those to render the legend with the right color values.

Reply:
Will do!  Do you want them in hex RGB format?